dataactcore/config_example.yml

```diff
@@ -68,6 +68,10 @@ broker:
     # AWS credentials for uploading files. Ignored if use_aws is false.
     aws_create_temp_credentials: true
 
+    # S3 filenames for SF-133 file, only required if planning to load SF-133 table
+    sf_133_folder: config
+    sf_133_file_one: sf_133_part1.csv
+    sf_133_file_two: sf_133_part2.csv
```

Comment:
Oh, it could certainly break Excel. But since the intent is to machine-load into the database, I don't think we should optimize for the use case that a human could open large .csv files in our S3 bucket.